runner/lib/filter/mod.rs

```diff
@@ -1,27 +1,93 @@
 use std::fs::{File};
-use std::path::Path;
 use std::env;
+use std::string::String;
+use crate::Error;
 use seccompiler::{compile_from_json, BpfProgram, TargetArch};
 
 const FILTERS_CONFIG_PATH: &str = "config/filters.json";
-
-pub fn build_filter() -> Result<BpfProgram, &'static str> {
-    // TODO: read json path from env variable, change default one
-    let config_path = match env::var("FILTERS_CONFIG_PATH") {
+/// # Reads filters from specified config file
+/// config file is to be found in following priority:
+/// 1. Read from CWA_FILTERS_CONFIG_PATH env variable
+/// 2. Read at config/filters.json
+pub(crate) fn build_filter() -> Result<BpfProgram, Error> {
+    // TODO: change default json_path
+    let preset_name = String::from("default");
+    let config_path = match env::var("CWA_FILTERS_CONFIG_PATH") {
         Ok(env_path) => { 
             env_path 
         },
         Err(_) => {
             String::from(FILTERS_CONFIG_PATH)
         }
     };
-    assert!(Path::new(config_path.as_str()).exists());
-    let mut filters = compile_from_json(
-        File::open(config_path.as_str()).unwrap(), 
-        TargetArch::x86_64
-    ).expect("Cannot compile filters from json");
+    let mut filters;
+    match File::open(config_path.as_str()) 
```

Comment:
`config_path` contains a relative path. When I run this from backend it fails, on current develop code at least, where there's an `assert!`

Reply:
I suppose backend should have a copy of the config too?